### ISSUE TITLE
Filename conflicts with composite primary keys

### ISSUE THREAD

**Issue by tillilab** [modmore/Gitify#218]
With some classes it happens that the same file name is used for different situations, 
example:

templateId: 4
tmplvarid: 15

will have same name as situation with:

templateId: 41
tmplvarid: 5

as a result just the second situation is saved into gitify.

It could be useful to add a key separator or something like that, 
thanks!

**Comment by Mark-H** [comment]
Doesn't it put a hyphen in between the different parts of a composite key?

**Comment by tillilab** [comment]
no, in the example above I have 415.yaml, but may be I'm missing something in configuration..
